SUPPORT_POLICY.rst

```diff
@@ -22,13 +22,13 @@ This table describes the current support status of each major version of the AWS
       - Next status
       - Next status date
     * - 1.x
-      - Maintenance
       - End of Support
-      - 2022-07-08
+      - 
+      - 
     * - 2.x
-      - Maintenance
       - End of Support
-      - 2022-07-15
+      - 
+      - 
     * - 3.x
       - Generally Available
       -
```

Comment:
Suggestion/Blocking: Add the maintenance date. 
This is the soonest we will move it to maintenance, 
6 months from now.
We can miss this this date, and often do. 
But we are supposed to give customer's at least 6 months warning. 
```suggestion
    * - 3.x
      - Generally Available
      - Maintenance
      - 2023-07-23
```